packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/template.success.tsx

```diff
@@ -254,7 +293,32 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
             </GreyBlueCartridge>
           </Amounts>
         )}
-        <MethodSelect {...props} />
+        <JumpToPayment
```

Comment:
👍

Reply:
You need also:

{props.method ? <ShowMethodHere {...props.method} /> : <JumpToPayment ... />}

For when the user has selected a method and you return them to this step, or if there is a default method passed in.

Reply:
yeah, I'll add it later have to see how and what of data I have to use there